valkyrie.drush.inc

```diff
@@ -121,7 +121,11 @@ function valkyrie_drush_command() {
     'aliases' => array('vup'),
     'bootstrap' => DRUSH_BOOTSTRAP_DRUSH, // No bootstrap at all.
   );
-
+  $items['valkyrie-upgrade'] = array(
```

Comment:
because we're doing lots of other updating and upgrading, I'd suggest 'valkyrie-self-upgrade' or '...-self-update'.